packages/app-content-pages/next.config.js

```diff
@@ -5,13 +5,7 @@ const Dotenv = require('dotenv-webpack')
 const path = require('path')
 const withSourceMaps = require('@zeit/next-source-maps')()
 
-const assetPrefixes = {
-  development: '/about',
-  branch: 'https://fe-project-branch.preview.zooniverse.org/about',
-  staging: 'https://frontend.preview.zooniverse.org/about',
-  static: 'https://fe-static.zooniverse.org/about',
-  production : 'https://www.zooniverse.org/about'
```

Comment:
Noting that depending on how this rebases after #4238 merges, and if we're in bypass CDN mode, this line should be `production : 'https://fe-project.zooniverse.org/about'` until the related issue is resolved.